testutil/network/network.go

```diff
@@ -106,7 +106,7 @@ func DefaultConfig() Config {
 		panic(fmt.Sprintf("failed creating temporary directory: %v", err))
 	}
 	defer os.RemoveAll(dir)
-	app := app.NewEvmos(log.NewNopLogger(), dbm.NewMemDB(), nil, true, nil, dir, 0, simutils.NewAppOptionsWithFlagHome(dir), baseapp.SetChainID(chainID))
+	app := app.NewEvmos(log.NewNopLogger(), dbm.NewMemDB(), nil, true, nil, dir, 0, simutils.NewAppOptionsWithFlagHome(dir), app.NoOpConfigurator, baseapp.SetChainID(chainID))
```

Comment:
why is the `NoOpConfigurator` the default? Don't we need the expected values to be set?

Reply:
no more relevant here